WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift

```diff
@@ -273,11 +274,32 @@ private class AccountSettingsController: SettingsController {
             case .failure(let error):
                 SVProgressHUD.dismiss()
                 DDLogError("Error closing account: \(error.localizedDescription)")
-                self?.showErrorAlert(message: error.localizedDescription)
+                let message = NSLocalizedString(self.generateLocalizedMessage(error), comment: "Error message displayed when unable to close user account.")
```

Comment:
I'm not sure if Localizable.strings will pick up the error messages if we pass generateLocalizedMessage as a param?

Reply:
It won't, we need to use the actual strings inside `NSLocalizedString`.